README.md

```diff
@@ -1 +1,37 @@
-# auto-semver-tag
\ No newline at end of file
+# Auto semver tag
+
+Action creates semver tag *(vX.Y.Z)* when pull request is merged. New version is calculated based on PR labels: "major", "minor". Patch version is incremented by default.
```

Comment:
```suggestion
Action creates semver tag *(vX.Y.Z)* when pull request is merged. New version is calculated based on presence PR labels: "major", "minor". Otherwise patch version is incremented.
```